webview-ui/src/CreateCluster/CreateCluster.module.css

```diff
@@ -2,13 +2,85 @@
     display: grid;
     grid-template-columns: 8rem 20rem 8rem;
     grid-gap: 1rem;
+    margin-top: 18px;
+}
```

Comment:
💭🙈🙏 If we can find some resource really chill to kindly do css help or see what else better we can do will be awesome.

Reply:
Thanks for the suggestions, I will look more into this.

Reply:
This is take care in the PR - https://github.com/hsubramanianaks/vscode-aks-tools/pull/4